### PR TITLE
created html boilerplate for Halogen

### PR DESCRIPTION
**Pull request by SioneMcKnight**
Hey, Sean here. I completed this. Thank you sir :]

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Halogen</title>
+</head>
+
+<body>
```
Comment (cbear1988uk) [inline index.html:10]: nice body

File: index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Halogen</title>
```
Comment (mikelbrierly) [inline index.html:7]: Hey dude! Good work! I think design-wise we want to keep the halogen name all lowercase. Do you think the title should be lowercase too?

image

Comment (SioneMcKnight) [reply index.html:7]: Good idea <3

**Review by mikelbrierly — APPROVED** [review]
Looks good! We'll take a look at the title change later 💯

**Pull request merged**